GUI/battle_menu.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import imgui
+
+from GUI.GUI import Window
+from GUI.menu import Menu
+from memory.combat_manager import CombatManager
+
+logger = logging.getLogger(__name__)
+
+combat_manager = CombatManager()
+
+
+class BattleMenu(Menu):
+    def __init__(self, window: Window) -> None:
+        super().__init__(window, title="Battle menu")
+
+    def execute(self, top_level: bool) -> bool:
+        self.window.start_window(self.title)
+        combat_manager.update()
+        # 4 seems to be the max enemies and players on screen
+        # This may change outside the demo
+        imgui.columns(4)
```

Comment:
Suggestion: make the 4 here (and the two instances below) into a named constant on the BattleMenu class (could also be a parameter to init, but probably not needed).